recipes/ffmpeg/all/conanfile.py

```diff
@@ -198,12 +198,15 @@ def _target_arch(self):
 
     @property
     def _target_os(self):
-        _, _, target_os = tools.get_gnu_triplet(
-            "Macos" if tools.is_apple_os(self.settings.os) else str(self.settings.os),
-            str(self.settings.arch),
-            str(self.settings.compiler) if self.settings.os == "Windows" else None,
-        ).split("-")
-        return target_os
+        if self.settings.os == "Windows":
```

Comment:
```suggestion
        if self.settings.os == "Windows" and self.settings.compiler == "Visual Studio":
```

Reply:
There is `intel` compiler (icl) that requires `win32` as well. maybe `        if "Visual Studio" in [str(self.settings.compiler), self.settings.compiler.get_safe("base")]:`?

Reply:
if it's target-os, it should be `win32` regardless of the compiler. same value for Visual Studio, Intel, Clang, Borland, etc.

Reply:
actually checking https://github.com/FFmpeg/FFmpeg/blob/9dd104f6e263dce770541074d99b5286524f2a8b/configure#L5382-L5640
it should be either `win32` or `win64`, so you should check arch here

Reply:
> if it's target-os, it should be win32 regardless of the compiler. same value for Visual Studio, Intel, Clang, Borland, etc.

no, at least for mingw it's different:

https://github.com/FFmpeg/FFmpeg/blob/9dd104f6e263dce770541074d99b5286524f2a8b/configure#L5478-L5479

Reply:
> it should be either win32 or win64, so you should check arch here

they're handled equally so win32 for both x86 and x86_64 is enough